### PR TITLE
Make the footer inline with the Mad specs.

### LINKED ISSUE #634: [footer]: on mobile make the spacing inline with the specs.
On the specs from MAD, the footer seems to have less spacing at the bottom than it is currently on the site. 
image

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #634 

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/640"></a>

### CODE REVIEW

File: src/components/footer.svelte [hunk 1]
```diff
@@ -19,7 +19,7 @@
   ];
 </script>
 
-<div class="footer__container pb-8">
```
Comment (nisarhassan12) [inline src/components/footer.svelte:22]: I have to remove this which was added in https://github.com/gitpod-io/website/commit/635723b94b108091b699e267e7ec389285e61e61#diff-c1498c6fdb1e64f27d6cec656dc0dd538adc5fda9f182c4e38a3ca65888b6d43 to fix the overlapping issue with the cookie consent. I re-added `pb-8` conditionally via JS only when the cookie consent is shown in the changes to  `src/components/banners/base.svelte` made in this Pr. @mikenikles I should probably move these to the `cookie-consent` file? right ?  Thanks.

File: src/components/banners/base.svelte [hunk 2]
```diff
@@ -4,16 +4,26 @@
...
 
   onMount(() => {
     showBanner = !window.localStorage.getItem(storageKey);
+    if (isTheCookieConsent) {
+      if (showBanner) {
+        document.querySelector(".footer__container").classList.add("pb-8");
+      } else {
+        document.querySelector(".footer__container").classList.remove("pb-8");
+      }
+    }
```
Comment (mikenikles) [inline src/components/banners/base.svelte:26]: This feels overly complex, especially combined with the code on lines 13 and 14. My comment on line 7 also applies here, this code should not be in a generic component that is used for top and bottom banners.

File: src/components/footer.svelte (hunk 1 above)
Comment (mikenikles) [reply src/components/footer.svelte:22]: If it's related to the cookie consent, yes the `cookie-consent.svelte` file is likely a better place.

**Review by mikenikles — CHANGES_REQUESTED** [review]
Thanks for looking into that Nisar. I understand the importance of matching the design spec, but in this case I don't think it's worth the additional complexity this PR introduces.

The `src/components/banners/base.svelte` component is a lot more complex now than it used to be and for someone to understand what happens in the `onMount()` lifecycle method, they have to look at the git history, find this PR and then the linked issue.

I would like to close this PR and leave the spacing the way it currently is in production.

**Comment by nisarhassan12** [comment]
> I would like to close this PR and leave the spacing the way it currently is in production.

Thanks. @mikenikles I agree with this let's close the PR.

**Pull request closed without merging**